bom/build.gradle.kts

```diff
@@ -0,0 +1,55 @@
+description = "BOM (Bill Of Materials) for graphql-kotlin"
+
+plugins {
+    `maven-publish`
+    `java-platform`
+}
+
+javaPlatform {
+    allowDependencies()
+}
+
+dependencies {
+    project.rootProject.subprojects.forEach { subproject ->
+        if (subproject.name != "graphql-kotlin-bom") {
+            api(subproject)
+        }
+    }
+}
+
+publishing {
+    publications {
+        create<MavenPublication>("graphql-kotlin-bom") {
+            from(components["javaPlatform"])
+            pom {
+                description = "BOM (Bill Of Materials) for graphql-kotlin"
+                name = "graphql-kotlin-bom"
+                packaging = "pom"
+                url.set("https://github.com/ExpediaGroup/graphql-kotlin")
+                licenses {
+                    license {
+                        name.set("The Apache Software License, Version 2.0")
+                        url.set("http://www.apache.org/licenses/LICENSE-2.0.txt")
+                    }
+                }
+                organization {
+                    name.set("Expedia Group")
+                    name.set("https://www.expediagroup.com/")
+                }
+                developers {
+                    developer {
+                        name.set("Expedia Group Committers")
+                        email.set("oss@expediagroup.com")
+                        organization.set("Expedia Group")
+                        organizationUrl.set("https://www.expediagroup.com/")
+                    }
+                }
+                scm {
+                    connection.set("scm:git:git://github.com/ExpediaGroup/graphql-kotlin.git")
+                    developerConnection.set("scm:git:git://github.com/ExpediaGroup/graphql-kotlin.git")
+                    url.set("https://github.com/ExpediaGroup/graphql-kotlin")
+                }
```

Comment:
These are defined in [com.expediagroup.graphql.conventions.gradle.kts](https://github.com/ExpediaGroup/graphql-kotlin/blob/d038e2e4aa8229e53e24c9046aa16c0c1a2378a6/buildSrc/src/main/kotlin/com.expediagroup.graphql.conventions.gradle.kts#L78-L112) and other java modules are using `id("com.expediagroup.graphql.conventions")` plugin.  But that plugin is a common module for java libraries, not applicable for this BOM.  copy-create with `pom` packaging.